### PR TITLE
Feature/2677: Apply Import Validation Checks (DTO) for Monitor Location ATTR Data to Import API

### PR DESCRIPTION
**Pull request by subrataroyCVP**
### Feature/2677: Apply Import Validation Checks (DTO) for Monitor Location ATTR Data to Import API

ZenHub Ticket #[2677](https://app.zenhub.com/workspaces/emissioners-scrum-board-5f36cd263511ad001a777f8e/issues/us-epa-camd/easey-ui/2677)

### CODE REVIEW

File: src/dtos/monitor-attribute-base.dto.ts [hunk 1]
```diff
@@ -37,34 +39,68 @@ export class MonitorAttributeBaseDTO {
...
+  })
   stackHeight: number;
 
   @ApiProperty({
     description: propertyMetadata.monitorAttributeDTOMaterialCode.description,
     example: propertyMetadata.monitorAttributeDTOMaterialCode.example,
     name: propertyMetadata.monitorAttributeDTOMaterialCode.fieldLabels.value,
   })
+  @IsInDbValues(
+    'SELECT distinct material_cd as "value" FROM camdecmpsmd.material_code',
```
Comment (jeffreyaggrey) [inline src/dtos/monitor-attribute-base.dto.ts]: Remove distinct

File: src/dtos/monitor-attribute-base.dto.ts [hunk 2]
```diff
@@ -37,34 +39,68 @@ export class MonitorAttributeBaseDTO {
...
+  )
   materialCode: string;
 
   @ApiProperty({
     description: propertyMetadata.monitorAttributeDTOShapeCode.description,
     example: propertyMetadata.monitorAttributeDTOShapeCode.example,
     name: propertyMetadata.monitorAttributeDTOShapeCode.fieldLabels.value,
   })
+  @IsInDbValues(
+    'SELECT distinct shape_cd as "value" FROM camdecmpsmd.shape_code',
```
Comment (jeffreyaggrey) [inline src/dtos/monitor-attribute-base.dto.ts]: Remove distinct

**Review by jeffreyaggrey — COMMENTED** [review]
Review comments

File: src/dtos/monitor-attribute-base.dto.ts (hunk 1 above)
Comment (subrataroyCVP) [reply src/dtos/monitor-attribute-base.dto.ts]: ✔

File: src/dtos/monitor-attribute-base.dto.ts (hunk 2 above)
Comment (subrataroyCVP) [reply src/dtos/monitor-attribute-base.dto.ts]: ✔

**Review by jeffreyaggrey — APPROVED** [review]

**Pull request merged**